CONTRIBUTING.md

```diff
@@ -1,47 +1,51 @@
 ## Development Instructions
 ### Setup
+
 Get the code!
 ```bash
-$ git clone https://github.com/cognitedata/cognite-sdk-python.git
-$ cd cognite-sdk-python
+git clone https://github.com/cognitedata/cognite-sdk-python.git
+cd cognite-sdk-python
 ```
 Install dependencies and initialize a shell within the virtual environment.
+
+To get the virtual environment, install [pipenv](https://pypi.org/project/pipenv/). Then execute:
+
 ```bash
-$ pipenv shell
-$ pipenv sync -d
+pipenv shell
+pipenv sync -d
 ```
 Install pre-commit hooks
 ```bash
-$ pre-commit install
+pre-commit install
 ```
 
 ### Environment Variables
 Set the following environment variables in a .env file:
 ```bash
-COGNITE_CLIENT_NAME = python-sdk-integration-tests
-COGNITE_MAX_RETRIES = 20
+COGNITE_CLIENT_NAME=python-sdk-integration-tests
+COGNITE_MAX_RETRIES=20
 
 # Only necessary for running integration tests
-COGNITE_PROJECT = python-sdk-test
-COGNITE_API_KEY = <api-key>
-COGNITE_BASE_URL = https://greenfield.cognitedata.com
+COGNITE_PROJECT=python-sdk-test
+COGNITE_API_KEY=<api-key>
+COGNITE_BASE_URL=https://greenfield.cognitedata.com
 ```
 
 ### Testing
 Set up tests for all new functionality. Running the tests will require setting the environment variable 'COGNITE_API_KEY'.
 
 Initiate unit tests by running the following command from the root directory:
 
-`$ pytest tests/tests_unit`
+`python3 -m pytest tests/tests_unit`
 
 If you have an appropriate API key, you can run the integratino tests like this:
 
-`$ pytest tests/tests_integration`
+`python3 -m pytest tests/tests_integration`
 
 If you want to generate code coverage reports run:
 
 ```
-pytest tests/tests_unit --cov-report html \
+python3 -m pytest tests/tests_unit --cov-report html \
```

Comment:
do you don't need `python3 -m` if you run it inside of `pipenv`?

Reply:
Oh, I was switching terminals because of another project and I think I executed tests in a terminal without pipenv. So you're correct.